cms/templates/js/settings-tab-section.underscore

```diff
@@ -0,0 +1,10 @@
+<ul class="settings-tab">
+    <li >
+        <a id="general_settings" href="#"><%- gettext('General Settings') %></a>
```

Comment:
@chrisndodge , the tabs are currently labelled "General Settings" and "Additional Settings". Can you suggest some better titles for the tabs? Moreover, these tabs only appear if either the Proctoring or Timed settings are set to true in Advanced settings.